### PR TITLE
#132 - Updates to config for contracts testing

### LINKED ISSUE #132: Simplifications + Testing for Juror contracts
Go through changes to update round robin handling to go through the jurors in the order that they have staked and keep track of the last juror that was selected.

Add testing to ensure that everything related to juror selection is working.

### PR DESCRIPTION
**Pull request by jtse0**
Close #132

### CODE REVIEW

File: apps/contracts/hardhat.config.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import 'hardhat-contract-sizer'
 import 'solidity-coverage'
 import 'tsconfig-paths/register'
 
-import { HardhatUserConfig } from 'hardhat/types'
+import { HardhatUserConfig } from 'node_modules/hardhat/types'
```
Comment (solomondefi-dev) [inline apps/contracts/hardhat.config.ts:12]: Why is this change needed? It seems to run on my machine with the previous version.

Comment (solomondefi-dev) [reply apps/contracts/hardhat.config.ts:12]: @jtse0 ping!

Comment (jtse0) [reply apps/contracts/hardhat.config.ts]: I've checked this issue and it seems like it should work with the previous versions so I have reverted this accordingly.  Updated in the latest changes.

**Comment by solomondefi-dev** [comment]
Looks like it's missing a lot of formatting, but I'm worried that running the formatter will cause a lot of conflicts with the other PR.

I'll merge now and we can take a look at CI errors once you rebase #158
